src/ParseQuery.js

```diff
@@ -16,6 +16,8 @@ import ParseGeoPoint from './ParseGeoPoint';
 import ParsePolygon from './ParsePolygon';
 import ParseObject from './ParseObject';
 import ParsePromise from './ParsePromise';
+import LocalDatastore from './LocalDatastore';
+import { matchesQuery } from 'parse-server/lib/LiveQuery/QueryTools';
```

Comment:
@flovilmart I think this like is my problem. I want to use this file. I get the following error.

```
GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected token: keyword (const)
File: /Users/Squad/Github/Parse-SDK-JS/dist/parse.js
Line: 16699
```

Looks like that file doesn't go through babel.

Should I just copy the files over or do you know of a better way.